src/BW/Vkontakte.php

```diff
@@ -318,6 +332,29 @@ public function getResponceType()
     }
 
     /**
+     * Set option enable for persistent connection
+     * @param boolean $enable
+     *
+     * @return $this
+     */
+    public function setPersistentConnect($enable)
+    {
+        $this->persistentConnect = (boolean)$enable;
+
+        return $this;
+    }
+
+    /**
+     * Get the status of type connection
+     *
+     * @return string
+     */
+    public function getPersistentConnect()
```

Comment:
I would like to use `isPersistentConnect` name for boolean types. Could you change?